src/main/java/fr/uge/chargepointconfiguration/chargepoint/ocpp/ocpp2/BootNotificationResponse.java

```diff
@@ -0,0 +1,16 @@
+package fr.uge.chargepointconfiguration.chargepoint.ocpp.ocpp2;
+
+import fr.uge.chargepointconfiguration.chargepoint.ocpp.OcppMessage;
+import fr.uge.chargepointconfiguration.chargepoint.ocpp.RegistrationStatus;
+
+/**
+ * Defines a response to a BootNotification message.
+ *
+ * @param currentTime String.
+ * @param interval An int.
```

Comment:
what is the meaning of this int ?

Reply:
Defines the interval between two BootNotificationRequest sent by the machine.
Will add this for javadoc

Reply:
Added [418f5d6](https://github.com/LastProject-ESIEE/OCPPConfigurationServer/pull/24/commits/418f5d60aaa3566fe2f766cb1ab37295e731dadc)